splitcomms

```diff
@@ -0,0 +1,54 @@
+- KernelCmdline
+- FirmwareFacts,
+- GrubDevice
+
+
+
+
+
+------------------------
+checkgrubcore??
+getinstalleddekstops (Installdekstopfacts)
+openssh* ??
+
+grubdevname & spol (remove? deprecated)
+
+-- missing intrdrefactorings, .... wairing for PR merge..
+
+-----------------------------
+tasks to generalize actors:
+  - checkleftovers
+  - checkosrelease
+  - RHUI (checkrhui refactoring, ....)
+  - enablerhsmreposonrhel8
+  - firewalldfacts - ebtables out.... contact eric
+  - multipathconfread??
+  - reportsettargetrelease (msgs)
+  - setuptargetrepos
+  - ipuconfig
+  - desktop - keep in common? (kde does not exist); gnome-session on el8
+
+- SAPHANA scanner: @vincenz is it working/valid on el8?
```

Comment:
From what I know it's generic enough to work. But we can give it a try and fix it in a follow up.